redaxo/src/core/assets/standard.js

```diff
@@ -653,4 +653,22 @@ jQuery(document).ready(function($) {
             ((windowHeight - rect.bottom) < menuHeight) &&
             (rect.top > menuHeight));
     });
+
+    // add eye-toggle to each password input
+    $('input[type="password"]').each(function () {
+        var $el = $(this);
+        var $eye = jQuery('<i class="fa fa-eye" aria-hidden="true"></i>');
```

Comment:
@tbaddade  ich hab mal das markup minimiert und jetzt in der standard.js an alle passwort felder gepackt.

kannst du dir bitte das markup selbst und das css mal dazu anschauen und anpassen..? aktuell hängt das auge ein bissl in der luft ;-).

ggf. kann man es genauso machen wie die symbole im be login (die vorm eingabefeld stehten) - nur eben dahinter...?

Reply:
@staabm 
Anstatt
`<i class="fa fa-eye" aria-hidden="true"></i>`

Müsste das notiert werden. Das Eye Icon erscheint dann bereits
`<i class="rex-icon rex-icon-view" aria-hidden="true"></i>`

Wegen weiterem Markup. Der richtige Aufbau wäre wie im Screenshot, der das Login zeigt.
![screenshot 2017-11-06 09 22 06](https://user-images.githubusercontent.com/338267/32431502-09c510e6-c2d4-11e7-87e5-94213b6dd7fb.png)

Jetzt gibt es aber Passwortfelder die bereits das `.input-group` mitbringen, andere Felder aber nicht. Entweder setzen wir das via Javascript oder wir passen das HTML direkt bei den Feldern an.

Dort wo das Schlossicon nicht zu sehen ein soll, wäre das Markup dann so:
![screenshot 2017-11-06 09 25 11](https://user-images.githubusercontent.com/338267/32431623-8292275c-c2d4-11e7-87aa-8e84943de03e.png)

Einzig die Klasse `.btn-view` müsste ich noch liefern. Der Rest würde schon korrekt dargestellt.

Reply:
super, ich schau am Freitag wieder danach.

Reply:
hab mal nachgebessert.. (ungetestet, sorry).

kannst du mal schauen obs passt?